collections/_posts/2023-01-31-NeRF.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: review
+title: "NeRF: Representing Scenes as Neural Radiance Fields for View Synthesis"
+tags: deep-learning scene-representation view-synthesis
+author: "Maylis Jouvencel"
+cite:
+    authors: "Ben Mildenhall, Pratul P. Srinivasan, Matthew Tancik, Jonathan T. Barron, Ravi Ramamoorthi, Ren Ng"
+    title:   "NeRF: Representing Scenes as Neural Radiance Fields for View Synthesis"
+    venue:   "ECCV 2020"
+pdf: "https://arxiv.org/pdf/2003.08934.pdf"
+---
+
+# Notes
+* Link to code, full paper and more examples in the [project page](https://www.matthewtancik.com/nerf)
+
+# Highlights
```

Comment:
Maybe it would be nice to add some context because I think that most of the participants will not be familiar with this topic =)